packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -89,7 +89,7 @@ import { isSfdxProjectOpened } from './predicates';
 import { registerPushOrDeployOnSave, sfdxCoreSettings } from './settings';
 import { taskViewService } from './statuses';
 import { telemetryService } from './telemetry';
-import { isCLIInstalled } from './util';
+import { getRootWorkspacePath, isCLIInstalled } from './util';
```

Comment:
Could we undo the change of this file since we don't use `getRootWorkspacePath` from core now?

Reply:
yes, thanks, I forgot to do that when I refactored the core dependency stuff, thanks for reminding me. See the `fix(soql): clean up unused exports and comment` commit